### DISCUSSION TITLE
Scroll amount

### DISCUSSION THREAD

**Discussion by DanielYorwarth** [idiotWu/smooth-scrollbar#329] [Q&A] [answered]
Is it possible to adjust the scrolled amount for each scroll?

I understand the damping controls the speed but is there a value I can change to adjust the scrolled amount. 

I'd like each scroll to be slightly shorter

See: https://katiforner.com/ 

Each mouse scroll seems to be about 50% of the default scroll

**Comment by dolphin-wood** [comment, marked as answer]
See https://github.com/idiotWu/smooth-scrollbar/issues/112#issuecomment-339814607
